ui/app/templates/partials/transit-form-show.hbs

```diff
@@ -28,38 +40,60 @@
   </nav>
 </div>
 
-<Toolbar>
-  <ToolbarActions>
-    {{#if (eq tab 'versions')}}
-      <TransitKeyActions
-        @key={{key}}
-        @selectedAction="rotate"
-        @capabilities={{capabilities}}
-        @onRefresh={{action "refresh"}}
-      />
-    {{/if}}
-    {{#if (eq mode "show") }}
-      {{#if (or capabilities.canUpdate capabilities.canDelete)}}
-        <ToolbarSecretLink
-          @secret={{key.id}}
-          @mode="edit"
-          replace=true
-        >
-          Edit encryption key
-        </ToolbarSecretLink>
+{{#unless (eq tab 'actions')}}
+  <Toolbar>
+    <ToolbarActions>
+      {{#if (eq tab 'versions')}}
+        <TransitKeyActions
+          @key={{key}}
+          @selectedAction="rotate"
+          @capabilities={{capabilities}}
+          @onRefresh={{action "refresh"}}
+        />
+      {{/if}}
+      {{#if (eq mode "show") }}
+        {{#if (or capabilities.canUpdate capabilities.canDelete)}}
+          <ToolbarSecretLink
+            @secret={{key.id}}
+            @mode="edit"
+            replace=true
+          >
+            Edit encryption key
+          </ToolbarSecretLink>
+        {{/if}}
       {{/if}}
-      <ToolbarSecretLink
-        @secret={{key.id}}
-        @mode="actions"
-        @data-test-transit-key-actions-link=true
-      >
-        Key actions
-      </ToolbarSecretLink>
-    {{/if}}
-  </ToolbarActions>
-</Toolbar>
+    </ToolbarActions>
+  </Toolbar>
+{{/unless}}
 
-{{#if (eq tab 'versions')}}
+
+{{#if (eq tab 'actions')}}
+  <div class="transit-card-container">
```

Comment:
I see you have both `transit-card.scss` imported as a component scss file, you don't have a TransitCard or TransitCardContainer component, you are just using the classes for styling.  I can see the argument for not turning them into components, but I would expect the scss file to be associated with components only because it's imported as one.  However, I also may not understand our conventions for how we determine how to import scss files in the core.scss file.

Reply:
this is a great thing to bring up! i opted to make this a style-only component (that is, i did not create a `TransitCardContainer` or `TransitCard` javascript file) because the component didn't need any logic associated with it. another example of a style-only component is here: https://github.com/hashicorp/vault/blob/master/ui/app/styles/components/vlt-table.scss